service/Microsoft.Azure.Devices.Uwp/Microsoft.Azure.Devices.Uwp.csproj

```diff
@@ -11,11 +11,12 @@
     <AssemblyName>Microsoft.Azure.Devices</AssemblyName>
     <DefaultLanguage>en-US</DefaultLanguage>
     <TargetPlatformIdentifier>UAP</TargetPlatformIdentifier>
-    <TargetPlatformVersion>10.0.10240.0</TargetPlatformVersion>
-    <TargetPlatformMinVersion>10.0.10240.0</TargetPlatformMinVersion>
+    <TargetPlatformVersion>10.0.15063.0</TargetPlatformVersion>
+    <TargetPlatformMinVersion>10.0.15063.0</TargetPlatformMinVersion>
```

Comment:
>15063 [](start = 35, length = 5)

is this required?

Reply:
I'll look more into this - I was getting some odd errors if I wasn't updating. 
Also, we should avoid support of SDKs that are too old to be useful and instead focus on the latest SDKs that can be used with UWP, including the latest Windows IoT Fall Update.